x-pack/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -60,6 +60,7 @@ export class VectorStyle extends AbstractStyle {
 
   constructor(descriptor = {}, source, layer) {
     super();
+    descriptor = descriptor === null ? {} : descriptor;
```

Comment:
Is this needed since function arguments defaults descriptor to `{}`

Reply:
a `null` value doesn't fallback to the default